gateway-service/src/main/java/org/zowe/apiml/gateway/websocket/ApimlWebSocketClient.java

```diff
@@ -10,23 +10,43 @@
 
 package org.zowe.apiml.gateway.websocket;
 
+import jakarta.websocket.ClientEndpointConfig;
 import jakarta.websocket.Session;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.tomcat.websocket.Constants;
 import org.apache.tomcat.websocket.WsWebSocketContainer;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.web.reactive.socket.HandshakeInfo;
 import org.springframework.web.reactive.socket.adapter.StandardWebSocketSession;
 import org.springframework.web.reactive.socket.client.StandardWebSocketClient;
 import reactor.core.publisher.Sinks;
 
+import java.util.List;
+
+@Slf4j
 public class ApimlWebSocketClient extends StandardWebSocketClient {
 
-    public ApimlWebSocketClient() {
-        super(new WsWebSocketContainer());
+    @Value("${server.webSocket.connectTimeout:45000}")
+    private String ioTimeout = "45000";
```

Comment:
```suggestion
    @Value("${server.webSocket.connectTimeout:45000}")
    private int ioTimeout = 45000;
```
Why it is string?

Reply:
because Tomcat expects:  String timeoutValue = (String) userProperties.get(Constants.IO_TIMEOUT_MS_PROPERTY);

Reply:
I guess we can use rather String.valueOf(n), but ok..